### PR TITLE
:hammer: walkthrough wipeout

### PR DESCRIPTION
**Pull request by Marigold**
Deprecate walkthrough in favour of the new wizard.

### CODE REVIEW

File: apps/walkthrough/utils.py
Comment (lucasrodes) [inline apps/walkthrough/utils.py:1]: feels like we may want to remove this file too?

Comment (Marigold) [reply apps/walkthrough/utils.py:1]: You're right. I did it thoroughly this time and wiped out all walkthrough mentions (removing `apps/walkthrough` as part of it).

**Comment by lucasrodes** [comment]
@Marigold I haven't looked at it, so I think it might not work in `wizard`. I think we should definitely have that, though. I can add it in the upcoming weeks (unless you do that first).

**Comment by Marigold** [comment]
> I think we should definitely have that, though. I can add it in the upcoming weeks (unless you do that first).

No worries, I don't need it now. I just though you could find it useful in case you find yourself filling metadata over and over again.

**Comment by Marigold** [comment]
@lucasrodes are we ready for this?

**Comment by lucasrodes** [comment]
hey @Marigold, I think we are almost good to go. Thanks for the reminder.

I'd say let me ask early next week if other people in the data team are still using walkthrough for some reason/miss something from it.

**Pull request merged**